comment_scan/comment_scanner.rb

```diff
@@ -21,13 +24,38 @@ def scan_new_comments
         new_comments = @seclient.comments_after_date(@latest_comment_date)
         @latest_comment_date = new_comments[0].json["creation_date"].to_i+1 if new_comments.any? && !new_comments[0].nil?
         scan_se_comments([new_comments])
+        check_for_hot_post([new_comments])
     end
 
     def tick
         (@time_to_scan = 60; return false) if (@time_to_scan -= 1) <= 0
         return true
     end
 
+    def check_for_hot_post(new_comments)
+        #Only check each post once, so make unique by post ID
+        new_comments.flatten.uniq(&:post_id).each do |comment|
+            continue unless post = @seclient.post_exists?(comment.post_id) # If post was deleted, skip it
+            comments_on_post = Comment.
+                                    where(post_id: comment.post_id).
```

Comment:
As a purely stylistic thing, I prefer the dots before the method names so that it's clear that those are methods being called on an object, not their own things.